tests/integration/questionnaire/test_questionnaire_progress_value_source_blocks.py

```diff
@@ -0,0 +1,113 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.questionnaire import SUBMIT_URL_PATH
+
+
+class TestQuestionnaireProgressValueSourceBlocks(IntegrationTestCase):
+    def test_skip_condition_block_not_complete(self):
```

Comment:
When I test this, if I answer do not answer 0 to the first question, then get to the summary and change my answer to the first question to 0, when I get back to the summary it looks like Question 4 is still on the path even though it should have been skipped? As question two is no longer on the path - is that correct?